extensions/arc/src/ui/dashboards/postgres/postgresComputeAndStoragePage.ts

```diff
@@ -500,9 +532,9 @@ export class PostgresComputeAndStoragePage extends DashboardPage {
 		return flexContainer;
 	}
 
-	private handleOnTextChanged(component: azdata.InputBoxComponent): boolean {
-		if ((!component.value)) {
-			// if there is no text found in the inputbox component return false
+	private handleOnTextChanged(component: azdata.InputBoxComponent, originalValue: string): boolean {
```

Comment:
Would be good to indicate in a doc comment what the return value indicates. Could also rename the function to make it clearer